### DISCUSSION TITLE
2024-08-16 - Creating lasting habits - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by bradystroud** [SSWConsulting/SSW.Rules.Content#9017] [Chewing the Fat]
https://www.ssw.com.au/rules/lasting-habits/

What are some tiny habits you could implement to improve your life?
Are there any apps or tools you use to help build and track habits?

**Comment by tiagov8** [comment]
Everyday I aim to close my **exercise**, **move**, and **stand** goals on Apple Fitness app. I've doing it for years and it's a great tool that encourages doing more exercises. 

<img src="https://github.com/user-attachments/assets/10ba8278-1cd4-41fb-8418-ed8828abda48" width="400">

**Comment by drwharris** [comment]
I need to start doing more than just sitting at my desk for 10 hours a day.

**Comment by bahjatalaadelssw** [comment]
I'm going to take up the flossing after brushing my teeth. My dentist is concerned 🫠

**Comment by vladislav-kir** [comment]
> What are some tiny habits you could implement to improve your life?

Exercising regularly would be great.

**Comment by ncn-ssw** [comment]
> What are some tiny habits you could implement to improve your life?

I'm not looking to add any habits to my life. 🤷

> Are there any apps or tools you use to help build and track habits?

Having someone get upset at you when you fail to perform a task that must be periodically conducted works well, but I'm not going to recommend it.

**Comment by jakebayliss** [comment]
> What are some tiny habits you could implement to improve your life?

Stretching more

**Comment by calumjs** [comment]
Tiny habits might make you feel good (and who knows, maybe work for some people), but I think if you really want to change something you need to go all out

**Reply by jakebayliss** [reply]
Whos gonna carry the boats

**Comment by ricksu978** [comment]
> What are some tiny habits you could implement to improve your life?

Stay healthy and make it become part of my life style.

> Are there any apps or tools you use to help build and track habits?

Fitbit App to track sleeps, walks and runs.

**Comment by isaaclombardssw** [comment]
> What are some tiny habits you could implement to improve your life?

Get through books more quickly by building reading into my routine.

**Comment by awaltos** [comment]
Apps are already doing some of this for me. E.g. Duolingo pushes me to do a small bit of work a couple times a day, my watch reminds me to get up and stretch, etc.

**Comment by uly1** [comment]
i use recurring reminders in my phone to make sure i do things that I commit to. e.g. weekly pushups, monthly checkins with my family in england, sending out the SSW Times, etc

**Comment by TobyChurches** [comment]
> What are some tiny habits you could implement to improve your life?
 
Would be nice to do more exercise after work
 
 > Are there any apps or tools you use to help build and track habits?
 
 The simple MS Todo app would work fine

**Comment by Calinator444** [comment]
> What are some tiny habits you could implement to improve your life?

- Taking my vitamins after breakfast
- Packing more food when I come to work so that I don't undereat
 
> Are there any apps or tools you use to help build and track habits?

I occasionally use this browser extension to block social media apps when I am trying to block out time to get something done but keep getting distracted.

https://chromewebstore.google.com/detail/stayfocusd-block-distract/laankejkbhbdhmipfmgcngdelahlfoji?utm_source=google_ads&utm_campaign=US&gad_source=1&gclid=Cj0KCQjwzva1BhD3ARIsADQuPnVi0t9iKgiQ-jCfOBSZa4gmK6d2pNqOGphfI0hrGWAtI5uScoIfIAEaAsMLEALw_wcB

**Comment by jeoffreyfischer** [comment]
**After I** am assigned the Imp 
**I will** pretend that I'm not the Imp

**Comment by Ben0189** [comment]
What are some tiny habits you could implement to improve your life?
rope skipping every morning 

Are there any apps or tools you use to help build and track habits?
Alarms

**Comment by leahy268** [comment]
>What are some tiny habits you could implement to improve your life?
When I finish work I will do 1 piece of yard work
Before dinner on Saturday and Sunday I will go for a walk

>Are there any apps or tools you use to help build and track habits?
No need for apps

**Comment by duanxinhuan** [comment]
What if tiny habits are reversed?
Like every time I wake up, I drink a can of Coke.

**Comment by bettybondoc** [comment]
> What are some tiny habits you could implement to improve your life?

After I finish my lunch break, I will use my standing desk for 15 minutes.
After I send my daily scrum email, I will delete an email.

> Are there any apps or tools you use to help build and track habits?

[Pomfocus](https://pomofocus.io/) - for Pomodoro
[Finch](https://apps.apple.com/us/app/finch-self-care-pet/id1528595748) - for Habit Tracking (it's like Tamagochi so it's fun)

**Comment by danielmackay** [comment]
> What are some tiny habits you could implement to improve your life?

when finishing working, I will leave my phone in my office so that I can be more present with my family

> Are there any apps or tools you use to help build and track habits?

Pomodoro
Outlook

**Comment by matt-goldman-ssw** [comment]
>What are some tiny habits you could implement to improve your life?

Drinking water and doing 10 pushups rather than snacking


>Are there any apps or tools you use to help build and track habits?

Not really, I wasn't able to get into the habit of using the app

**Comment by jaydenalchin** [comment]
Don't underestimate the power of a recurring reminder or calendar event that keeps your phone buzzing at you

**Comment by chrisschultzssw** [comment]
> What are some tiny habits you could implement to improve your life?

I guess I'll start flossing 🙃

> Are there any apps or tools you use to help build and track habits?

Not at the moment, no

**Comment by miichaelsmedley** [comment]
dentist - flossing - regular doctor visits

**Comment by christoment** [comment]
The age is quickly catching up - I should start doing exercise regularly 😄

**Comment by Dhruv-0987** [comment]
What are some tiny habits you could implement to improve your life?
Sleeping earlier than usual, I feel like all my problems will go away if it sleep for 8-9 hours everyday
 
Are there any apps or tools you use to help build and track habits?
nope, never works well for me

**Comment by sethdaily** [comment]
My favorite one is when I drink water I drink that amount of water again

**Comment by ozair-ashfaque** [comment]
> What are some tiny habits you could implement to improve your life?

Exercise

> Are there any apps or tools you use to help build and track habits?

None

**Comment by Tanya-Leahy** [comment]
What are some tiny habits you could implement to improve your life?

Drink more water and snack less. Exercise more.

Are there any apps or tools you use to help build and track habits?

I am going to start using the Samsung Health app.

**Comment by jimmy1992abc** [comment]
>What are some tiny habits you could implement to improve your life?

After I check the chalk on my car (every 2 hours), I need to have a glass of water 

>Are there any apps or tools you use to help build and track habits?

When you have bad habits, think of something bad.
E.g, when not drinking enough, think about dehydrating and Calculus

**Comment by AttackOnMorty** [comment]
> Are there any apps or tools you use to help build and track habits?

- [Zero](https://apps.apple.com/us/app/zero-fasting-health-tracker/id1168348542)
- [Streaks](https://streaksapp.com/)
- [Be Focused](https://apps.apple.com/us/app/be-focused-focus-timer/id973134470)
- [Calflow](https://apps.apple.com/us/app/calflow-calendar-time-tracker/id6474122188)

**Comment by MattParkerDev** [comment]
> What are some tiny habits you could implement to improve your life?

Exercise more

> Are there any apps or tools you use to help build and track habits?

I don't use any currently

**Comment by Marxoz** [comment]
Aren't you supposed to floss BEFORE you brush? Anyway....

One of the techniques I learned a while ago is sort of related to this. If you have something you need to do, and it will take less than a minute, do it immediately

**Comment by jerryluossw** [comment]
> What are some tiny habits you could implement to improve your life?

After sitting for 1 hour of work, stand and work for 15 minutes.

> Are there any apps or tools you use to help build and track habits?

Scheduling management apps, I believe there are many options available. I personally use the app that comes pre-installed on my Xiaomi phone.

**Comment by babakamyljanovssw** [comment]
> 1. What are some tiny habits you could implement to improve your life?

Start morning with exercises

> 2. Are there any apps or tools you use to help build and track habits?

I don't use any app

**Comment by jimmidier** [comment]
> What are some tiny habits you could implement to improve your life?

Work out more and stay energetic

> Are there any apps or tools you use to help build and track habits?

None

**Comment by Aibono1225** [comment]
> What are some tiny habits you could implement to improve your life?

Read more, jog more, drink (water) more

> Are there any apps or tools you use to help build and track habits?

Keep - for jogging

**Comment by KristenHu** [comment]
> What are some tiny habits you could implement to improve your life?

Exercise more.
>Are there any apps or tools you use to help build and track habits?

I haven't tried currently

**Comment by tino-liu** [comment]
> What are some tiny habits you could implement to improve your life?

Clean up more and exercise more

> Are there any apps or tools you use to help build and track habits?

Apple's Reminder

**Comment by StefStar** [comment]
Exercise more and read more!

**Comment by PennyWalker** [comment]
I'm going to drink more water by having a glass everytime I have a cup of tea (which is a lot)

**Comment by stephanfakossw** [comment]
> What are some tiny habits you could implement to improve your life?
Being more active while at work
